pandas/tools/pivot.py

```diff
@@ -382,7 +382,9 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
     columns : array-like, Series, or list of arrays/Series
         Values to group by in the columns
     values : array-like, optional
-        Array of values to aggregate according to the factors
+        Array of values to aggregate according to the factors.
```

Comment:
this is the same thing as having ``aggfunc`` default to ``'count'``, so let's just make that explict if that's the case. Though I think it should just be an error like it was. Why are you changing this?